ui/scss/component/_wunderbar.scss

```
@@ -97,22 +83,50 @@
+    width: 1rem;
+    height: 1rem;
+  }
+
+  &:hover {
+    background-color: var(--color-odysee);
   }
 }
 
 .wunderbar__input--inline {
   @extend .wunderbar__input;
+  border-radius: var(--border-radius);
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)